IntelliTect.TestTools.WindowsTestWrapper/ExampleProjects/Notepad.AutomatedTests/Tests/ExampleNotepadTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.VisualStudio.TestTools.UITesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+
+namespace Notepad.AutomatedTests.Tests
+{
+    [CodedUITest]
+    public class ExampleNotepadTest : BaseTest
+    {
+        //These may be turned into data drive tests for conciseness and flexibility
+        [TestMethod]
+        [TestCategory("ExampleTest")]
+        public void NoCharactersEntered()
+        {
+            NotepadWindow.NotepadEdit.Text = Empty;
+            NotePadWindowHelpers.SaveDocument(SaveLocation, "testNoCharacters.txt");
+            var text = Utilities.ParseTextFile(SaveLocation + @"\testNoCharacters.txt");
+            Assert.AreEqual(Empty, text, "Unexpected text was found");
+            Utilities.DeleteDocument(SaveLocation, "testNoCharacters.txt");
```

Comment:
I am undecided. I considered putting the DeleteDocument call in the test teardown, which will run even if there's a failure, but I wasn't sure if there would be value in being able to inspect the file should there be a failure. Same as above: I'll do some poking around with this to see if I can determine if one way is better than the other.